csp.ipynb

```diff
@@ -969,18 +961,18 @@
    "metadata": {},
    "source": [
     "We have defined our CSP. \n",
-    "We now need to solve this.\n",
+    "Now, we need to solve this.\n",
     "\n",
     "### Min-conflicts\n",
     "As stated above, the `min_conflicts` algorithm is an efficient method to solve such a problem.\n",
     "<br>\n",
-    "To begin with, all the variables of the CSP are _randomly_ initialized. \n",
+    "In the start, all the variables of the CSP are _randomly_ initialized. \n",
```

Comment:
I think 'To begin with...' sounds better, but its up to you.

Reply:
Agreed, it sounds better, but the meaning that needs to be conveyed at this point requires "In the start", as i believe you wish to tell the audience that in the start you initialized them randomly and later on you changed/trained them. I hope you get my point. Thank you for reviewing.

Reply:
Sure, it's fine either way

Reply:
@ad71 is there a way to integrate the changes you suggested (to reverse) within this PR?

Reply:
Just add another commit (in the same branch) with the changes you want

Reply:
Alright, perfect. Thank you.